main.go

```diff
@@ -109,15 +115,15 @@ func main() {
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```

Comment:
Just fixing the logger setup that I got into the wrong place during earlier refactoring.